scripts/utils.ts

```diff
@@ -0,0 +1,64 @@
+import * as fs from 'fs';
+import fetch from 'node-fetch';
+
+export interface Obj {
+    [key: string]: string;
+}
+
+export const checkFile = (fileName: string) => {
+    return new Promise((resolve ,reject) => {
+        fs.access(fileName, (err) => {
+            if (err) return resolve(null);
+
+            return reject(new Error("File does exist!"));
+        })
+    })
+}
+
+export const writeFile = (filePath: string, fileContent: string) => {
+    return new Promise((resolve, reject) => {
+        fs.writeFile(filePath, fileContent, (err) => {
+            if (err) return reject(err);
+
+            return resolve(null);
+        })
+    }) 
+}
+
+export const appendFile = (filePath: string, fileContent: string) => {
+    return new Promise((resolve, reject) => {
+        fs.appendFile(filePath, fileContent, (err) => {
+            if (err) return reject(err);
+
+            return resolve(null);
+        });
+    });
+}
+
+export const readFile = (filePath: string): Promise<string> => {
+    return new Promise((resolve, reject) => {
+        fs.readFile(filePath, 'utf-8', (err, content) => {
+            if (err) return reject(err);
+
+            return resolve(content);
+        })
+    })
+}
+
+
+export const getAbiPath = (answers: Obj) => `src/abi/connectors/v${answers.version}/${answers.name}.ts`;
+export const getAbiDir = (answers: Obj) => `src/abi/connectors/v${answers.version}`;
+export const addressV1Path = `src/addresses/mainnet/connectorsV1.ts`;
+export const addressV2Path = `src/addresses/mainnet/connectorsV2_M1.ts`;
```

Comment:
@DakshMiglani Also, let's add support for polygon?

Reply:
I thought of doing that, but are polygon ABIs available on etherscan?

Reply:
Nope etherscan doesn't support etherscan. So, we need to see polygon explorer. https://maticvigil.com/docs/